src/test/shell/bazel/bazel_cc_code_coverage_test.sh

```diff
@@ -100,37 +115,26 @@ EOF
   cat << EOF > "$ROOT_VAR/coverage_srcs/t.cc"
 #include <stdio.h>
 #include "a.h"
+#include "different/a.h"
 
 int main(void) {
   a(true);
+  different_a(false);
 }
 EOF
 
   generate_and_execute_instrumented_binary coverage_srcs/test \
-      coverage_srcs/a.h coverage_srcs/a.cc \
-      coverage_srcs/b.h \
-      coverage_srcs/t.cc
-
-  # Prior to version 11, g++ generates the notes files in the current directory
-  # instead of next to the object file despite the documentation indicating otherwise:
-  # https://gcc.gnu.org/onlinedocs/gcc/Gcov-Data-Files.html#Gcov-Data-Files
-  # This is fixed in g++ 11 so we have to handle both cases.
-
-  local not_found=0
-  ls coverage_srcs/*a.gcno > /dev/null 2>&1 || not_found=$?
-  if [[ $not_found -ne 0 ]]; then
-    agcno=$(ls *a.gcno)
-    tgcno=$(ls *t.gcno)
-    agcda=$(ls *a.gcda)
-    tgcda=$(ls *t.gcda)
-    mv $agcno coverage_srcs/$agcno
-    mv $tgcno coverage_srcs/$tgcno
-    mv $agcda coverage_srcs/$agcda
-    mv $tgcda coverage_srcs/$tgcda
-  fi
+      coverage_srcs/a.cc coverage_srcs/a.o \
+      coverage_srcs/different/a.cc coverage_srcs/different/a.o \
+      coverage_srcs/t.cc coverage_srcs/t.o
+
+  find .
```

Comment:
Should drop the find; it clutters the logs when there are failures.

Reply:
Done